source/mainnet/smart-contracts/tutorials/eSealing/eSealing_dapp.rst

```diff
@@ -0,0 +1,89 @@
+.. include:: ../../../variables.rst
+.. _eSealing_dapp:
+
+=============
+eSealing dApp
+=============
+
+You can explore the `hosted eSealing service <https://esealing.testnet.concordium.com>`_ or
+start your own front-end by following the instructions in the READ.me file of this
+`repository <https://github.com/Concordium/concordium-browser-wallet/tree/main/examples/eSealing>`_ to get an overview of the dApp flow.
+
+.. note::
+
+   Comprehensive instructions on how to set up the |bw|, create an account in the |bw|,
+   get some testnet CCD and run a local front-end can be found in :ref:`wCCD front-end-set-up section <wCCD-front-end-set-up>`
+
+The front end supports the following two flows with the |bw| (or |mw-gen2| that uses WalletConnect):
+
+- Compute the file hash of a selected file => register its file hash in the smart contract
+- Compute the file hash of a selected file => retrieve the timestamp and witness (sealer account) that registered the file hash
+
+.. note::
+
+   Select the ``registration tab`` to register the file hash and the ``display tab`` to look up the
+   timestamp and witness (sealer account) of an already registered file.
+
+.. note::
+
+   A file hash is an algorithmic way to assign a unique string to a file using a cryptographic hash function (e.g., sha256) based on the file content.
+   Changing the file's content would result in a completely new file hash. The eSealing smart contract expects a file hash of length 256-bits as an
+   input parameter because of the used `HashSha2256 <https://docs.rs/concordium-std/latest/concordium_std/struct.HashSha2256.html>`_ type.
```

Comment:
This link seems to be at the wrong level of abstraction for this tutorial. I would suggest just referring to the wiki link I suggest above for the number of bits.

Reply:
I feel that because `HashSha2256` is a custom type, this link is useful (e.g. the first sentence "Sha2 digest with 256 bits (32 bytes)." was very helpful when I first checked our documentation).